packages/shared_preferences/shared_preferences_windows/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.0.3-nullsafety
```

Comment:
The Windows changes can be reverted, as there is already a PR in progress at https://github.com/flutter/plugins/pull/3516

Reply:
Alright, doing it.